provision/acc_provision/flavors.yaml

```diff
@@ -159,10 +159,39 @@ flavors:
     default_version: 1.9
     status: null
     hidden: true
-    order: 50
+    order: 51
     config:
       kube_config:
         no_priority_class: True
+  openshift-sdn-ovn-baremetal:
+    desc: Red Hat OpenShift Container Platform 4.12 on Baremetal
```

Comment:
Pl change to:
Chained CNI for Red Hat OpenShift Container Platform 4.12 on Baremetal

Reply:
Resolved